src/styles/carousel.scss

```diff
@@ -153,48 +143,36 @@ $button: #{$fd-namespace}-button;
     text-align: center;
   }
 
-  &__button {
-    &.#{$button} {
-      @include fd-flex-center();
-      @include set-width($fd-carousel-button-size);
-      @include set-height($fd-carousel-button-size);
-
-      border-radius: 50%;
-      padding: 0;
-      box-shadow: none;
-      margin: 0.25rem;
-
-      &::before {
-        left: -0.25rem;
-        right: -0.25rem;
-        height: $fd-carousel-touchable-button-size;
-        width: $fd-carousel-touchable-button-size;
-      }
-
-      @include fd-rtl() {
-        & > [class*=sap-icon] {
-          transform: rotate(180deg);
-        }
-      }
+  .#{$block}__button {
+    @include fd-flex-center();
+    @include set-width($fd-carousel-button-size);
+    @include set-height($fd-carousel-button-size);
 
-      @include fd-focus() {
-        @include after-position-overwrite();
-      }
+    border-radius: 50%;
+    padding: 0;
+    margin: 0.25rem;
+
+    &::before {
+      left: -0.25rem;
+      right: -0.25rem;
+      bottom: -0.25rem;
+      top: -0.25rem;
```

Comment:
Can't this value go to a local variable? In future if it changes we will need to update it in 4 places.